src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts

```diff
@@ -311,7 +311,11 @@ describe("backendFromV1Alpha1", () => {
           },
         },
       };
-      const expected = backend.of({ ...DEFAULTED_ENDPOINT, scheduleTrigger });
+      const expected = backend.of({
+        ...DEFAULTED_ENDPOINT,
+        scheduleTrigger,
+        labels: { "deployment-scheduled": "true" },
```

Comment:
This LOC is why we add the labels in `functionFromEndpoint`, so we don't have any logic that breaks if tests are missing a label. DRY and make the one field (scheduleTrigger) mean the one thing (that this function is scheduled)